marketplace-client/src/marketplace/src/app/system-accounts/filter/filter.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { StaffMember } from 'src/app/_models/staff-member';
+import { UserDto } from 'src/app/_models/UserDto';
+import { SystemAccountService } from 'src/app/_services/system-account.service';
+
+@Component({
+  selector: 'app-filter',
+  templateUrl: './filter.component.html',
+  styleUrls: ['./filter.component.css'],
+})
+export class FilterComponent implements OnInit {
+  activeRadioButton = '';
+  users: StaffMember[] = [];
+  @Input() status_list: string[] = [];
+
+  //@Output() results: EventEmitter<User[]> = new EventEmitter<User[]>();
+  @Output() results: EventEmitter<any> = new EventEmitter<any>();
+
+  constructor(private service: SystemAccountService) {}
+
+  ngOnInit(): void {
+    this.activeRadioButton = this.getFilter();
+    this.filterStatus(this.activeRadioButton, true);
+  }
+
+  filterStatus(status: string, init: boolean) {
```

Comment:
all your class members are public. But some of them are used only inside of the class and might be made private